package.json

```diff
@@ -49,6 +49,7 @@
     "markdown-it-attrs": "^3.0.0",
     "markdown-it-container": "^2.0.0",
     "markdown-it-footnote": "^3.0.2",
+    "markdownlint": "^0.16.0",
```

Comment:
Is this necessary?

Reply:
Good catch, probably not, I upgraded `markdownlint` from command line, when it should have been `markdownlint-cli`.

Reply:
Actually... looks like it's already latest. Weird, as the rule should be disabled then.

**UPD**: Ah, the reason was that there was explicit `"MD002": ...` line in the config which re-enabled the rule, but now that it's removed, everything works as intended.

Reply:
Fixed.